src/chrome/content/rules/Ecwid.com.xml

```diff
@@ -0,0 +1,20 @@
+<!--
+	Non-functional hosts:
+		Incomplete certificate chain error:
+		- ideas.ecwid.com
```

Comment:
Looks like customers get their own subdomain and all of them support HTTPS

Reply:
Do you think adding a top comment on this would be sufficient to inform the future contributors? there is over 1k subdomains discovered by Sublist3r, and it will take significant time to test them all.

Reply:
Either this or a wildcard target.

Reply:
I would prefer to keep this ruleset as-is. Thank you.